### PR TITLE
Added packer image for AMI builds

### PR DESCRIPTION
**Pull request by hod-alpert**
## Status
Ready

## Related Issues
related: https://github.com/demisto/etc/issues/25490

## Description
Created am image with packer and it's dependencies

### CODE REVIEW

**Comment by glicht** [comment]
Is this only for development?

**Comment by hod-alpert** [comment]
> Is this only for development?

@glicht Yes

**Review by glicht — CHANGES_REQUESTED** [review]
If this is for dev only add to build.conf: `devonly=true`. Remove the license changes as this is dev only. Also please use Pipfile for installing python dependencies as we do in the other images.

File: docker/packer/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+RUN curl -so packer.zip $(curl -sk https://www.packer.io/downloads.html |grep -oE https://releases\.hashicorp\.com/packer/\[0-9.\]\{5\}/packer_\[0-9.\]\{5\}_linux_amd64\.zip)
+
+RUN unzip packer.zip
+
+RUN mv packer /usr/bin/
+
+RUN rm -f packer.zip
+
+RUN pip3 install jinja2 requests python-dateutil dateparser pyyaml ansible jmespath
```
Comment (glicht) [inline docker/packer/Dockerfile:15]: please change to use a Pipfile and Pipfile.lock as we do with other images.

File: docker/packer/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,24 @@
+FROM demisto/python3-deb:3.8.6.12176
+
+RUN apt-get update
+
+RUN apt-get install curl jq unzip openssh-client -y
+
+RUN curl -so packer.zip $(curl -sk https://www.packer.io/downloads.html |grep -oE https://releases\.hashicorp\.com/packer/\[0-9.\]\{5\}/packer_\[0-9.\]\{5\}_linux_amd64\.zip)
```
Comment (glicht) [inline docker/packer/Dockerfile:7]: Please use a specific version which is locked.

Comment (glicht) [reply docker/packer/Dockerfile:7]: Or if you want the latest include a printout of the version when done installing as we do with gcloud install.

**Review by glicht — APPROVED** [review]

**Pull request merged**